### PR TITLE
Set rspec_command only outside of tmate

### PR DESCRIPTION
**Pull request by christopher-haueter**

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -195,7 +195,9 @@ let g:spec_runner_dispatcher = 'call Send_to_Tmux("clear\n{command}\n")'
 let g:indentobject_meaningful_indentation = ['haml', 'sass', 'yaml', 'markdown']
 let g:indentLine_fileType = ['yaml']
 
-let g:rspec_command = 'call Send_to_Tmux("rspec {spec}\n")'
+if !system("tmate show-messages | grep tmate &> /dev/null")
```
Comment (arktisklada) [inline vimrc]: Could we also add a condition to be sure we're in tmux?  `if exists('$TMUX')` for example?

Comment (arktisklada) [inline vimrc]: Also, is it the error message you're testing against?  I found that $TMUX env var includes a long string, including tmate when it's tmate.  Just an idea to explore that may be a bit more robust depending on how you were intending to use this condition.

That string looks something like this:
`/private/var/folders/g9/9ckx387yx4z_xynh47bbn3n40000gn/T/tmate-501/SLZvIT,3627,0`

**Comment by christopher-haueter** [comment]
@arktisklada the new commit simultaneously checks that we're in tmux and not in tmate.

**Comment by hannahhearth** [comment]
:+1:

**Comment by hannahhearth** [comment]
This fixes my spec runner

**Comment by arktisklada** [comment]
@chrishaum nice!! :rocket:

**Pull request merged**